### PR TITLE
Add version check

### PR DESCRIPTION
**Pull request by mofeiwang**
Add version check for llat.
Current version as per @xinzhang-lotusflare  is 0.1

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -33,6 +35,11 @@ func stop(cCtx *cli.Context) error {
 	return nil
 }
 
+func get_version(cCtx *cli.Context) error {
```
Comment (xinzhang-lotusflare) [inline main.go:38]: Method name should be CamelCase

Make sure executing `go fmt` before committing

File: main.go [hunk 2]
```diff
@@ -33,6 +35,11 @@ func stop(cCtx *cli.Context) error {
...
+	fmt.Println("llat version", VERSION)
```
Comment (xinzhang-lotusflare) [inline main.go:39]: We can do

```
fmt.Println("llat version: ", VERSION)
```

or simply

```
fmt.Println(VERSION)
```

Just make sure the actual "version" is separated with the hints

Comment (mofeiwang) [reply main.go:39]: Make sense, will do,,

File: main.go (hunk 1 above)
Comment (mofeiwang) [reply main.go]: Done

File: main.go [hunk 3]
```diff
@@ -97,6 +104,11 @@ func main() {
 				},
 				Action: install,
 			},
+			{
+				Name:   "version",
+				Usage:  "Check current llat version",
+				Action: get_version,
```
Comment (xinzhang-lotusflare) [inline main.go:110]: We should modify this as well

Comment (mofeiwang) [reply main.go]: Done

**Review by xinzhang-lotusflare — APPROVED** [review]
LGTM

**Pull request merged**